all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p>Clean the long output before pushing if it's not crucial to show it.</p><p> </p><p>Also, use string formatting (see above).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>